x-pack/plugins/uptime/server/lib/alerts/tls.ts

```diff
@@ -128,27 +109,39 @@ export const tlsAlertFactory: UptimeAlertTypeFactory<ActionGroupIds> = (_server,
 
       const foundCerts = total > 0;
 
+      const uniqueCerts = certs.reduce<Cert[]>((uniqueCertArray, currentCert) => {
+        if (!uniqueCertArray.find((cert) => cert.common_name === currentCert.common_name)) {
+          uniqueCertArray.push(currentCert);
+        }
+
+        return uniqueCertArray;
+      }, []);
+
       if (foundCerts) {
-        const absoluteExpirationThreshold = moment()
-          .add(
-            dynamicSettings.certExpirationThreshold ??
-              DYNAMIC_SETTINGS_DEFAULTS.certExpirationThreshold,
-            'd'
-          )
-          .valueOf();
-        const absoluteAgeThreshold = moment()
-          .subtract(
-            dynamicSettings.certAgeThreshold ?? DYNAMIC_SETTINGS_DEFAULTS.certAgeThreshold,
-            'd'
-          )
-          .valueOf();
-        const alertInstance = alertInstanceFactory(TLS.id);
-        const summary = getCertSummary(certs, absoluteExpirationThreshold, absoluteAgeThreshold);
-        alertInstance.replaceState({
-          ...updateState(state, foundCerts),
-          ...summary,
+        uniqueCerts.forEach((cert) => {
+          const absoluteExpirationThreshold = moment()
+            .add(
+              dynamicSettings.certExpirationThreshold ??
+                DYNAMIC_SETTINGS_DEFAULTS.certExpirationThreshold,
+              'd'
+            )
+            .valueOf();
+          const absoluteAgeThreshold = moment()
+            .subtract(
+              dynamicSettings.certAgeThreshold ?? DYNAMIC_SETTINGS_DEFAULTS.certAgeThreshold,
+              'd'
+            )
+            .valueOf();
+          const alertInstance = alertInstanceFactory(
+            `${TLS.id}-${cert.common_name}-${cert.issuer}`
```

Comment:
i don't think we need `tls.id` common_name and issue should be enough. tls.id will pollute ui where we are displaying list of instances in alert management ui